index.html

```diff
@@ -6,190 +6,229 @@
+                <p>Manager;</p>
             </article>
             <article>
                 <h3>octomber, 2008 – september, 1994, State Protection and Guard Service Chisinău, Republic of Moldova
                 </h3>
-                <p>military service by contract</p>
+                <p>Military service by contract;</p>
             </article>
             <article>
                 <h3>september, 1994 – july, 1993, Institute of Applied Physics, Academy of Sciences of the Republic of
                     Moldova Chisinău, Republic of Moldova</h3>
-                <p>engineer in the semiconductor compound laboratory</p>
+                <p>Engineer in the semiconductor compound laboratory.</p>
             </article>
         </section>
 
-        <section>
+        <section id="skills">
             <h2>Software Development Skills</h2>
             <article>
                 <h3>Programming</h3>
                 <ul>
-                    <li>HTML5/CSS3</li>
-                    <li>JavaScript</li>
-                    <li>PHP</li>
-                    <li>MySQL</li>
+                    <li>HTML5/CSS3;</li>
+                    <li>JavaScript;</li>
+                    <li>PHP;</li>
+                    <li>MySQL.</li>
                 </ul>
             </article>
             <article>
                 <h3>Development tools</h3>
                 <ul>
-                    <li>Bootstrap 4.5</li>
-                    <li>GitHUB</li>
+                    <li>Bootstrap 4.5;</li>
+                    <li>GitHUB.</li>
                 </ul>
             </article>
             <article>
                 <h3>Visualisation, Publishing</h3>
                 <ul>
-                    <li>D3.js</li>
-                    <li>LaTex</li>
+                    <li>D3.js;</li>
+                    <li>LaTex.</li>
                 </ul>
             </article>
         </section>
 
-        <section>
+        <section id="languages">
             <h2>Languages</h2>
             <ol>
-                <li>English, fluent</li>
-                <li>Romanian, native</li>
-                <li>Russian, native</li>
+                <li>English, fluent;</li>
+                <li>Romanian, native;</li>
+                <li>Russian, native.</li>
             </ol>
         </section>
 
-        <section>
+        <section id="miscellaneous">
             <h2>Miscellaneous</h2>
             <p>Driver license - A, B, C, D, E</p>
         </section>
     </main>
 
     <footer>
-        <article id="contact">
-            <h3>Contact me</h3>
-            <p>Contact</p>
-        </article>
-        <article>
-            <h3>Mircea CĂLIN</h3>
-            <p>Software developer</p>
-        </article>
+        <section id="contact">
+            <h2>Contact me</h2>
+            <article>
+                <h3>Feel free to contact me</h3>
+                <form action="" method="get">
+                    <input type="text" name="name" placeholder="Name" /><br />
```

Comment:
Void elements only have a start tag; end tags must not be specified for void elements.
 https://html.spec.whatwg.org/multipage/syntax.html#void-elements
 
 In rest all is fine.